src/post/post.entity.ts

```diff
@@ -97,6 +97,7 @@ export class Post extends BaseEntity {
     private repostsToRemove: Set<number> = new Set();
     private deleted = false;
     public readonly content: string | null;
+    public readonly mentions: Account[] = [];
```

Comment:
I think we need to populate this when reading from the repository too- I'm imagining a situation where the `PostCreatedEvent` switches to storing a `postId` instead of a full `Post` in future (because we are sending over a queue, and need to limit bandwidth e.g.), then we call `getById` and everything looks good, but federated mentions are silently failing 😢

Reply:
Added changes in https://github.com/TryGhost/ActivityPub/pull/671/commits/ed90d4e675370e0ef3329eccdfc4437857e89c68 and https://github.com/TryGhost/ActivityPub/pull/671/commits/cb7bc1b340a7108a7870352c03a0737282ff78c6